### PR TITLE
Mahdiyeh/ remove_cryptocurrency_market

### PR DESCRIPTION
**Pull request by mahdiyeh-fs**

### CODE REVIEW

**Review by matin-binary — APPROVED** [review]

**Review by matin-binary — CHANGES_REQUESTED** [review]
The reason behind disabling cryptocurrencies is that **Crypto** only supports Multipliers in `contracts_for`, it is the same case for **Jump indices**. Since we don't have Multipliers in platforms other than Deriv-app, we are trying to hide based on the markets, but it is best to filter out based on shortcodes using regex to make sure it will handle other cases and avoid double work in future.
So I suggest using `\(?:JD[0-9])|(?:cry_[A-Z])\` to filter out shortcodes. `JD` for Jump indices and `cry` for Cryptos. 
Feel free to have a look into Zaki's changes in this PR https://redmine.deriv.cloud/issues/30826

**Review by matin-binary — CHANGES_REQUESTED** [review]
Please help to update the PR

**Comment by mahdiyeh-fs** [comment]
> regex

sure, I will work on that

File: src/navigation/menu.es6 [hunk 1]
```diff
@@ -50,8 +50,11 @@ export const extractChartableMarkets = (trading_times_data) => {
 };
 
 export const refreshMenu = (root, markets, callback) => {
+   const non_crypto = symbol => !/^(cry|JD)/i.test(symbol);
```
Comment (matin-binary) [inline src/navigation/menu.es6:53]: Please use a more descriptive name for this function!

Comment (mahdiyeh-fs) [reply src/navigation/menu.es6]: sure

**Review by matin-binary — APPROVED** [review]

**Pull request merged**